todo_app/app.py

```diff
@@ -1,11 +1,28 @@
-from flask import Flask
-
+from flask import Flask, url_for, render_template, request, redirect
+from todo_app.data.session_items import get_items, add_item
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
-@app.route('/')
+# routes the URL path
+@app.route('/', methods=["GET"])
 def index():
-    return 'Hello World!'
+    # renders index.html to the route '/' and passes the argument get_items() from session_items.py file
+    return render_template('index.html', get_items = get_items)
```

Comment:
Minor, but here you're passing a function that returns a list of items, rather than a list of items. It would be more expected to pass the list of items itself. This helps emphasise the difference between 'server-side' and 'client-side' functions.
```suggestion
    return render_template('index.html', items=get_items())
```